components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/APIResourceManager.java

```diff
@@ -156,4 +156,13 @@ void putScopes(String apiResourceId, List<Scope> currentScopes, List<Scope> scop
      * @throws APIResourceMgtException If an error occurs while retrieving scope.
      */
     Scope getScopeByName(String scopeName, String tenantDomain) throws APIResourceMgtException;
+
+    /**
+     * Get scope metadata by scope names and tenant domain.
+     *
+     * @param scopeNames   List of scope names.
+     * @param tenantDomain Tenant domain.
+     */
```

Comment:
Add exception to description

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5003/commits/ea4b23a67c37fdf0e9b8c39e8ee2dccd0167fbda